exasol/bucketfs/_buckets.py

```diff
@@ -223,6 +252,120 @@ def download(self, path: str, chunk_size: int = 8192) -> Iterable[ByteString]:
             yield from response.iter_content(chunk_size=chunk_size)
 
 
+class SaaSBucket:
+
+    def __init__(self, url: str, account_id: str, database_id: str, pat: str) -> None:
+        self._url = url
+        self._account_id = account_id
+        self.database_id = database_id
+        self._pat = pat
+
+    @property
+    def name(self) -> str:
+        # TODO: Find out the name of the bucket in SaaS
+        return 'default'
+
+    @property
+    def udf_path(self) -> str:
+        # TODO: Find out the name of the service in SaaS
+        # and how the udf path is constructed. Below is just a guess.
+        return f'buckets/bfsdefault/{self.name}'
```

Comment:
I asked @tomuben for the name of the bucketfs service and the bucket.
However, the path should be absolute

Reply:
from @tomuben

    bucketfs-name=uploads
    bucket-name=default

Reply:
Is the `/buckets/` in front still applicable?

Reply:
yes